article.html

```diff
@@ -0,0 +1,359 @@
+            speck of primordial energy to the size of a grapefruit — in less than a cosmic eye-blink.
+        </p>
+        <p>
+            Almost as quickly, this pulse would subside, relaxing into ordinary particles and radiation. All of 
+            normal cosmic history was still ahead, resulting in today’s observable universe, a patch of sky and stars billions 
+            of light-years across. “It’s often said that there is no such thing as a free lunch,” Dr. Guth likes to say, “but the 
+            universe might be the ultimate free lunch.”
+        </p>
+        <p>
+            Make that free lunches. Most of the hundred or so models resulting from Dr. Guth’s original 
+            vision suggest that inflation, once started, is eternal. Even as our own universe settled down to a 
+            comfortable homey expansion, the rest of the cosmos will continue blowing up, spinning off other bubbles 
+            endlessly, a concept known as the multiverse.
+        </p>
+        <p>
+            So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into the past.
+        </p>
+        <p>
+            We might never know what happened before inflation, at the very beginning, because inflation erases everything that came 
+            before it. All the chaos and randomness of the primordial moment are swept away, forever out of our view.
+        </p>
+        <p>
+            “If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”
+        </p>
+    </section>
+    <footer id="footer1">
+        A version of this article appears in print on March 18, 2014, Section A, Page 1 of the New York edition with the headline: Space Ripples Reveal Big Bang’s Smoking Gun. 
+        <a href="">Order Reprints</a> | <a href="">Today’s Paper</a> | <a href="">Subscribe</a>
+    </footer>
+         
+    </article>
+
+    <div id="soc_flex">
+        <button>Read Comments</button>
+        <div>
+        <a href=""><img class="social_icons" src="icons/fb-icon.png" alt="facebook"></a>
+        <a href=""><img class="social_icons" src="icons/twitter-icon.png" alt="twitter"></a>
+        <a href=""><img class="social_icons" src="icons/mail-icon.png" alt="mail"></a>
+        <a href=""><img class="social_icons" src="icons/share-icon.png" alt="share"></a>
+    </div>
+    </div>
+
+<footer id="footer2">
+    <a href="">© 2020 The New York Times Company</a>
+    <a href="">NYTCo</a>
+    <a href="">Contact Us</a>
+    <a href="">Work with us</a>
+    <a href="">Advertise</a>
+    <a href="">T Brand Studio</a>
+    <a href="">Your Ad Choices</a>
+    <a href="">Privacy Policy</a>
+    <a href="">Terms of Service</a>
+    <a href="">Terms of Sale</a>
+    <a href="">Site Map</a>
+    <a href="">Help</a>
+    <a href="">Subscriptions</a>
+</footer>    
```

Comment:
- [ ] [Optional] Please use a single footer here. You can then use the nested footer if you want them.